### ISSUE TITLE
how to reduce the size of the selected date circle?

### ISSUE THREAD

**Issue by deepakrana0011** [prolificinteractive/material-calendarview#633]
Hi , 
 i need some help. How can i reduce the selected date circle ?Please help me out.

**Comment by Vincebrees** [comment]
Change the decorator or add a specific decorator when you select a date. 
Read the documentation, watch the sample example, it will highly help you.

**Comment by iad24** [comment]
@Vincebrees Is it possible to have a simple property for this? It seems a lot of people wants this feature without adding the whole src code or creating a new decorator.

**Comment by Vincebrees** [comment]
Nope its not possible. This is Android, where everything is customizable, if we set a property for this, we lose customization. It take 1 hour to implement it, read the documentation it's quite easy.

**Comment by iad24** [comment]
yes, developers can always do the customized date selector they want. But to those who are contented with the current design but just need to reduce the size, this property would be helpful. 
We put the 1 hour code in this lib, and lot of developers will save that time by using the 1 second property.

**Comment by deepakrana0011** [comment]
thank you @Vincebrees

**Comment by smithshelke** [comment]
Do you realize how many times will the "should decorate" method be called. Every click will have a maximum of 42 "should decorate" calls.
No wonder why this library lacks performance.
